packages/nouns-webapp/src/components/Bid/index.tsx

```diff
@@ -247,13 +255,28 @@ const Bid: React.FC<{
             <span className={classes.customPlaceholder}>ETH</span>
           </>
         )}
-        <Button
-          className={auctionEnded ? classes.bidBtnAuctionEnded : classes.bidBtn}
-          onClick={auctionEnded ? settleAuctionHandler : placeBidHandler}
-          disabled={isDisabled}
-        >
-          {bidButtonContent.loading ? <Spinner animation="border" /> : bidButtonContent.content}
-        </Button>
+        {
+          !auctionEnded ? (
+              <Button
+                className={auctionEnded ? classes.bidBtnAuctionEnded : classes.bidBtn}
+                onClick={auctionEnded ? settleAuctionHandler : placeBidHandler}
+                disabled={isDisabled}
+              >
+                {bidButtonContent.loading ? <Spinner animation="border" /> : bidButtonContent.content}
+              </Button>
+          ) 
+          : (
+            <>
+              <Button
+                className={classes.bidBtnAuctionEnded}
+                onClick={fomoNounsBtnOnClickHandler}
+              >
+                Help mint the next Noun ⌐◨-◨  
+            </Button>
+            <p className={classes.emergencySettleWrapper}>In case of emergency, <button onClick={settleAuctionHandler} className={classes.emergencySettleButton}>settle the auction manually.</button></p>
```

Comment:
When a wallet isn't connected, the original "Settle Auction" button was disabled. Should the emergency settle button follow the same logic?

Reply:
+1 to this ... I'll add it ...

Reply:
Done. Made it so this copy only shows for connected wallets